Hindawi Publishers.js

```diff
@@ -80,7 +89,27 @@ function doWeb(doc,url)
 				// use article ID as a page (seems to be the last part of URL/DOI)
 				item.pages = 'e' + item.DOI.substr(item.DOI.lastIndexOf('/') + 1);
 			}
-			item.extra = "";
+			//remove duplicate metadata			
```

Comment:
No, it's for all fields that don't get overwritten when you call them twice. Their EM has every highwire field duplicated.